.github/workflows/prestonbot_on_pr.yml

```diff
@@ -0,0 +1,41 @@
+name: Prestonbot actions on PR
+
+on: [ pull_request ]
+
+env:
+  prestonbot_repository: PrestaShop/prestonbot
+  GH_TOKEN: ${{ secrets.JARVIS_TOKEN }}
+  GH_SECURED_TOKEN: ${{ secrets.JARVIS_TOKEN }}
+
+concurrency:
+  group: ${{ github.event_name }}-${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  dispatch_pr_in_prestonbot:
+    name: Dispatch PR in prestonbot
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout PrestonBot
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ env.prestonbot_repository }}
+          path: 'prestonbot/'
+          token: ${{ secrets.PS_GITHUB_TOKEN }}
```

Comment:
It seems ok for me but the CI seems to be blocking because of a missing token here, even though you specified it To investigate once the tool PR ill have been merged anyway since it's blocking for now